codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostRequestDecoderTest.java

```diff
@@ -1002,4 +1002,29 @@ public void testDecodeMultipartRequest() {
             assertTrue(req.release());
         }
     }
+
+    /**
+     * when diskFilename contain "\" create temp file error
+     */
+    @Test
+    void testHttpPostStandardRequestDecoderBySize(){
+        StringBuffer sb = new StringBuffer();
+        for (int i = 0 ;i<300 ;i++){
```

Comment:
If you set Factory to use disk, no need to make a big one. Just one is enough, so directly a String. No need of StringBuffer.

Reply:
The reason for using StringBuffer is When initializing the mixedattribute object with defaulthttpdatafactory，
Only when the request message size exceeds 16kb, the mixedattribute can level up DiskAttribute ，When upgrading to diskattribute, you need to create a temporary file  Therefore, a file creation exception will appear.

Reply:
I believe that what you're trying to do is to show that when the name is containing a "/", and when it generates a temporary file (and only if so), it will bring an exception due to bad path name.
Therefore, if you force the Factory to use disk, it will go immediately to the issue.
If you don't, using default Factory one, then if the size of each item is < 16KB, it will never go to your issue, therefore demonstrating nothing.

In your example, each item (badly written) is abour 4 bytes with a name of 9 bytes, event if you set a buffer of more than 16KB. I add a comment on the right way to set those items.